src/controllers/urlController.js

```diff
@@ -0,0 +1,68 @@
+import { UrlService } from '../service/urlService.js';
+import { UserService } from '../service/userService.js';
+
+export class UrlController {
+    constructor() {
+        this.urlService = new UrlService();
+        this.userService = new UserService();
+    }
+
+    async createShortUrl(request, reply) {
+        try {
+            const { user_id, url: original_url } = request.body;
+            const response = await this.urlService.createShortUrl(user_id, original_url);
+
+            if (response.status === 'error') {
+                return reply.status(400).send({
+                    message: response.message
+                });
+            }
+
+            return reply.status(201).send({
+                status: response.status,
+                short_url: response.short_url,
+                original_url: response.original_url

```

Comment:
realizar responses como `snake_case` não é o ideal em aplicações javascript, onde o mais comum é o `camelCase`. procurando um pouco, parecem ter formas facilitadas de realizar essa conversão utilizando knex ou o objection: https://vincit.github.io/objection.js/recipes/snake-case-to-camel-case-conversion.html